eng/targets/RazorSDKDirectoryOverride.props

```diff
@@ -0,0 +1,6 @@
+<Project>
+  <PropertyGroup>
+    <RazorSdkDirectoryRoot Condition="'$(RazorSdkDirectoryRootOverride)' != ''">$(RazorSdkDirectoryRootOverride)</RazorSdkDirectoryRoot>
```

Comment:
Allow tests to override this. The reason this is needed is that some tests build templates with the Debug configuration. However, given that the SDK along with tests are all built in the Release configuration, we need to override the property to ensure the SDK is resolved correctly.

Reply:
The `Condition` isn't useful here. Just set `$(RazorSdkDirectoryRoot)` to `$(RazorSdkDirectoryRootOverride)`.

Alternatively leave this line out entirely and set `$(RazorSdkDirectoryRoot)` where necessary. Right now, it looks like `$(RazorSdkDirectoryRootOverride)` is set as a global property when overridden anyhow.